core/src/main/scala/fs2/Async.scala

```diff
@@ -189,6 +189,13 @@ object Async {
    */
   case class Change[+A](previous: A, now: A)
 
+  object Change {
+    implicit class ChangeSyntax[A](val self:Change[A]) extends AnyVal {
```

Comment:
Yeah. I used to this approach because this guarantees there won't be any hidden state kept in case class. I have no strong preference on this, so lets target change of it with any other PR related to signal? Is that ok with you or you think is worth of separate PR?